client/lib/hoverIntent/README.md

```diff
@@ -0,0 +1,64 @@
+## Hoverintent
```

Comment:
Should this be a h1? Meaning, a single `#`

Reply:
+1. This is (or should be) already covered by 'Code Style' tests

This file is generating lots of Code Style warnings, it would be nice if we can fix them before merging it.